pipeline/usfimr_raster_pipeline.py

```diff
@@ -169,7 +170,7 @@ def build_dataset_from_catalog(catalog: Catalog, class_config: ClassConfig,
     return DatasetConfig(
         class_config=class_config,
         train_scenes=train_scenes,
-        validation_scenes=validation_scenes,
+        validation_scenes=(validation_scenes + test_scenes),
```

Comment:
I'm curious about the exclusion of a `test_scenes` here

Reply:
I am including `test_scenes` here.  I just want them to be part of raster vision's validation set.